ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WatchedImagesModal.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
When we start creating the inner components for this Modal, where do you imagine keeping those? From the directory structure, I think maybe having the watched images components in a separate directory within `WorkloadCVEs` would make sense. That way we can separate the new components from the overview components

Reply:
Sure, I can move this to a separate directory.